client/src/components/authentication/AuthenticatedRoute.js

```diff
@@ -17,9 +17,9 @@ const AuthenticatedRoute = ({ component, user, ...rest }) => {
       // eslint-disable-next-line react/jsx-props-no-spreading
       {...rest}
     >
-      <AuthenticationCheck user={user} component={component} />
+      <AuthenticationCheck user={user} component={component} {...rest} />
```

Comment:
we should not have to update/ modify this file, the `...rest` is being added to the Route as needed

Reply:
This was from Nick, he took control for a min